client/tests/RelatedProducts.test.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+
+import RelatedProducts from '../components/relatedProducts/RelatedProducts';
+
+describe('Related Products', () => {
+  const mockProductId = 43432;
+  const mockRatiing = 3.5;
+
+  beforeEach(() => {
+    render(<RelatedProducts productId={mockProductId} rating={mockRatiing} />);
```

Comment:
interestingly you used the same typo here so this probably worked

Reply:
At least I'm a consistent bad speller. 😜